styles/output/apbiology-pdf.css

```
@@ -586,22 +586,17 @@ nav#toc > .os-toc-title {
   color: #000000;
   font-family: Alegreya;
   font-size: 1.2rem;
-  line-height: 1.5rem;
-  font-weight: normal; }
+  line-height: 1.5rem; }
 
 .os-index-container > .group-by > .os-index-item > .os-term {
   color: #000000;
   font-family: Alegreya;
   font-size: 1.2rem;
   line-height: 1.5rem;
-  font-weight: normal;
   padding-right: 8px; }
 
 .os-index-container > .group-by > .os-index-item > .os-term-section-link {
-  content: "" target-counter(attr(href, url), page);
-  text-decoration: none;
-  pointer-events: none;
-  color: #000000; }
+  content: "" target-counter(attr(href, url), page); }
```

Might be outside the scope of this PR, but does
```suggestion
  content: target-counter(attr(href, url), page); }
```
work?

---

I removed this empty string today form the settings file, because it didn't affect the pdf layout in chemistry and pushed the changes.